dozer-types/src/ingestion_types.rs

```diff
@@ -184,6 +190,15 @@ pub struct SnowflakeConfig {
     pub driver: Option<String>,
 
     pub role: String,
+
+    #[serde(
+        default = "default_snowflake_poll_interval",
+        skip_serializing_if = "equal_default_snowflake_poll_interval",
+        deserialize_with = "deserialize_duration_secs_f64",
+        serialize_with = "serialize_duration_secs_f64"
+    )]
+    #[schemars(schema_with = "schema_for_duration_secs_f64")]
+    pub poll_interval_seconds: Duration,
```

Comment:
Let's use `Option<Duration>` (or `Option<f64>`, with all the `...with` it seems easier) instead.

The reason behind this is that we've experienced several bugs because of inconsistent `Default` value with the default deserialized value. Therefore we decided to remove all `#[serde(default = ...)]` in https://github.com/getdozer/dozer/pull/2078.

Reply:
I changed it to `Option<Duration>`. I didn't use `f64` directly because it doesn't implement `Eq` and `Hash`.